graph/path/internal/testgraphs/limited.go

```diff
@@ -166,6 +166,14 @@ func (l *LimitedVisionGrid) Has(id int64) bool {
 	return 0 <= id && id < int64(len(l.Grid.open))
 }
 
+// Node return a node from g if it exists.
```

Comment:
Not sure if we should mention node ID here? Right now its implicit from the context. Same question regarding `Grid.Node`.

Reply:
Fixed in both.